Apps/Inc/Contactor.h

```diff
@@ -17,11 +19,7 @@
 
 //These are defining how the contactors are wired, the array contactor is assumed to be wired to the fan board
 #define ARRAY_CONTACTOR_OUT CFAN
-#define HVHIGH_CONTACTOR_OUT C1
-#define HVLOW_CONTACTOR_OUT C1
-
-typedef uint8_t CONT_CHOICE;
-typedef enum {ARRAY_CONTACTOR = 1, HVHIGH_CONTACTOR = 2, HVLOW_CONTACTOR = 4, ALL_CONTACTORS = 7} Contactors_t;
+#define HV_CONTACTOR_OUT C1
```

Comment:
In the event that we have 3 contactors. I would like to still keep ``HVLOW_CONTACTOR_OUT`` and just have it equal to ``C1``